changelog.d/215.doc

```diff
@@ -0,0 +1 @@
+Add connection timeout error code  #215. Contributed by Omar Mohamed
```

Comment:
For some reason we really want these changelog entries to end with a . or ! mark -- please add one :)